docs/generated/packages/angular/generators/remote.json

```diff
@@ -163,6 +163,11 @@
         "description": "Whether to configure SSR for the remote application to be consumed by a host application using SSR.",
         "type": "boolean",
         "default": false
+      },
+      "typescriptConfiguration": {
```

Comment:
Why is this not called `useTypescriptConfiguration`? 

It feels like the field should point to the configuration file, but it's actually just a boolean flag.

I guess this applies to other fields above (ssr, standalone) as well. If we want to match the same style we could name this one `typescript`.